app/views.py

```diff
@@ -384,6 +390,22 @@ def get_context_data(self, **kwargs):
         return context
 
 
+def presto_query(request):
+    query_params = request.GET
+    date_format = "%Y-%m-%d"
```

Comment:
can you use `SUPERSET_QUERY_DATE_FORMAT` here too?

Reply:
great catch! missed it 👀